packages/openneuro-app/src/scripts/dataset/__tests__/__snapshots__/snapshot-container.spec.tsx.snap

```diff
@@ -786,14 +786,14 @@ OCI-1131441 (R. Poldrack, PI) in any publications.
             N/A
           </div>
           <div
-            class="dataset-meta-block dmb-inline-list"
+            class="dataset-meta-block "
           >
             <h2
               class="dmb-heading"
             >
-              Authors
+              Contributors
```

Comment:
Even if this is contributors in datacite.yml, we might want to keep the authors label here? BIDS calls it authors and datacite.yml is the one exception.

Reply:
updated in both draft/snapshot-container.tsx and snapshot